src/skin.cpp

```diff
@@ -245,16 +245,11 @@ void SkinInfillAreaComputation::applySkinExpansion(const Polygons& original_outl
  *
  * this function may only read/write the skin and infill from the *current* layer.
  */
-void SkinInfillAreaComputation::generateSkinInsets(SliceLayerPart* part)
+void SkinInfillAreaComputation::generateSkinInsetsAndInnerSkinInfill(SliceLayerPart* part)
```

Comment:
Rule of thumb: If a function has the word "and" in its name, it does two things! I think the generating of inner skin infill can quite easily be separated even here.